docs/podman-container-cleanup.1.md

```diff
@@ -0,0 +1,38 @@
+% podman-container-cleanup "1"
+
+## NAME
+podman\-container\-cleanup - Cleanup Container storage and networks
+
+## SYNOPSIS
+**podman container cleanup [OPTIONS] CONTAINER**
```

Comment:
I think you want to distinguish between the literal `podman container cleanup` and the placeholder `OPTIONS` and `CONTAINER`.  From [`man-pages(7)`][1]:

> For commands, this shows the syntax of the command and its arguments (including options); boldface is used for as-is text and italics are used to indicate replaceable arguments. Brackets ([]) surround optional arguments, vertical bars (|) separate choices, and ellipses (...) can be repeated.

So I recommend:

```markdown
+**podman container cleanup** [*options*] *container*
```

[1]: http://man7.org/linux/man-pages/man7/man-pages.7.html

Reply:
Ok lets do that in a follow on man page cleanup.  I don't want to hold this up from getting in.